### DISCUSSION TITLE
Hi, is it possible to do LoRA on an intel server without GPU?

### DISCUSSION THREAD

**Discussion by skytraveler** [zetavg/LLaMA-LoRA-Tuner#20] [Q&A]
I am looking fore  the possibility to LoRA on a Intel server without GPU, I wonder is it possible to do that.
there's almost no articles about this.
Could you give me some feedback? thank you.

**Comment by zetavg** [comment]
I believe that the performance of training on CPUs is uncompetitive with GPUs, and it might not be cost-effective to do so. Sorry about that, but I think I would recommend just to use GPUs. I personally use cloud GPUs on Lambda Cloud (A10), GCP spot VMs, and Google Colab.
